environments/production/inventory.ini.j2

```diff
@@ -176,6 +176,10 @@ shareddir1
 
 {{ __es31__ }} elasticsearch_node_name=es31
 
+{{ __es31__ }} elasticsearch_node_name=es31
```

Comment:
It looks like this is what's breaking the build. Can remove this line

Reply:
I have deleted branch Unfortunately and made another PR  https://github.com/dimagi/commcare-cloud/pull/4396 which is merged